COPYRIGHT

```diff
@@ -1,3 +1,6 @@
+The code in the "libcperciva/external" directory is distributed
```

Comment:
As with spiped, I'm not completely certain that you'll want the copyright notice here.